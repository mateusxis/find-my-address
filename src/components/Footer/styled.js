import styled from 'styled-components';

export const SFooter = styled.footer`
  display: flex;
  padding: 1rem 0;
  border-top: 1px solid #eaeaea;
  justify-content: center;
  align-items: center;
  color: #fff;
`;

export const SLink = styled.a`
  display: flex;
  justify-content: center;
  align-items: center;
  flex-grow: 1;
  cursor: pointer;

  &:hover {
    color: #95a5a6;
  }
`;
