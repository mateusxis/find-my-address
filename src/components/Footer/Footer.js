import { SFooter, SLink } from './styled';

const Footer = () => (
  <SFooter>
    <SLink href="https://github.com/mateusxis" target="_blank" rel="noopener noreferrer">
      Desenvolvido por Mateusxis
    </SLink>
  </SFooter>
);

export default Footer;
