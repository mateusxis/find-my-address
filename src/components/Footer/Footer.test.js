import renderer from 'react-test-renderer';

import Footer from './Footer';

describe('Footer', () => {
  it('should render Footer', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
