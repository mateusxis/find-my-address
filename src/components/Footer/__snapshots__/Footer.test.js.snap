// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer 1`] = `
<footer
  className="sc-bdvvtL iuZHJh"
>
  <a
    className="sc-gsDKAQ gsXWMv"
    href="https://github.com/mateusxis"
    rel="noopener noreferrer"
    target="_blank"
  >
    Desenvolvido por Mateusxis
  </a>
</footer>
`;
